services/webhook/deliver.go

```diff
@@ -202,17 +202,19 @@ func Deliver(ctx context.Context, t *webhook_model.HookTask) error {
 	return nil
 }
 
-// DeliverHooks checks and delivers undelivered hooks.
+// populateDeliverHooks checks and delivers undelivered hooks.
 // FIXME: graceful: This would likely benefit from either a worker pool with dummy queue
 // or a full queue. Then more hooks could be sent at same time.
```

Comment:
I think the comment should be adjusted:
```suggestion
// populateDeliverHooks populates the webhook queue with undelivered webhooks.
```